src/renderer/widgets/PasswordInputField.cpp

```diff
@@ -255,21 +291,27 @@ void CPasswordInputField::updateFailTex() {
             g_pRenderer->asyncResourceGatherer->unloadAsset(placeholder.failAsset);
             placeholder.failAsset = nullptr;
             placeholder.failID    = "";
+            redrawShadow          = true;
         }
         return;
     }
 
     if (!FAIL.has_value() || !placeholder.canGetNewFail)
         return;
 
+    if (placeholder.failText.empty())
+        placeholder.failText = "<span style=\"italic\">" + FAIL.value() + "</span>";
```

Comment:
instead of checking for emptiness, I'd just set this as the default. You can put `$FAIL` in the defaults.

Someone might want to (for some reason) set it to _actually_ empty.

Reply:
ye u're right